css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,98 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+
+  display: flex;
```

Comment:
I have added `{{sidebarclasses}}` in a pending PR, if this gets approved, we could look to move flex and align-items as basscss classes into ampstart-sidebar